### ISSUE TITLE
Event dragging not shown when excessive minTime/maxTime

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#3055]
from @mlaufer

> I just tried the negative minTime/more than 24h maxTime feature, which you implemented with #112 and there is a bug with FC.TimeGrid.Grid.renderHelper, more specifically with FC.TimeGrid.Grid.computeDateTop.
> 
> When you drag an event onto the negative minTime section or the "more than 24h" maxTime section, it seems like its not rendered, but its just the top and bottom values, which are 0.
> 
> Fiddle: http://jsbin.com/cixuwosibe/1/edit

**Comment by pcstro** [comment]
Hi! I have the same problem as described here.
When I'm trying to move an event past 12am or 24h, it disappears..
Using agendaDay-view and updated to the last fullcalendar version.
minTime are set to 08:00 and maxTime to 28:00.
Is there any fix or workaround for this?

**Comment by enekochan** [comment]
I'm having the same problem. The moment I drag or place an event beyond 24:00 it's given 0px to top and bottom so it disappears but it's still there (somewhere...):

```
<div class="fc-event-container">
    <a class="fc-time-grid-event fc-v-event fc-event fc-start fc-end ..."
        style="top: 0px; bottom: 0px; z-index: 1; left: 0%; right: 0%;">
        ....
    </a>
</div>
```

On the other side if I create an editable agendaView calendar, add an event that starts at lets say 22:00 and ends at 23:00, then dragging the end of the event over 24:00 works, it doesn't disapear and I can see the correct end datetime (for example 01:00 of the next day) on the console if I print it on eventClick.

![screen shot 2016-04-29 at 16 30 24](https://cloud.githubusercontent.com/assets/1836845/14919154/bd868276-0e27-11e6-9983-c9832d025283.png)

**Comment by enekochan** [comment]
The function returning 0 is `computeTimeTop`, this happens when the event start being rendered is beyond 24:00. As a VERY rough workaround @pcstro you can change `computeDateTop` function (that calls `computeTimeTop`) and check there if `computeTimeTop` returns 0, in that case recalculate adding 1 day (86400 sec. or 86400000 millis.). The problem is that this only works if your `minTime` and `maxTime` are 24 hours or less apart.

```
	computeDateTop: function(date, startOfDayDate) {

		...

		var cTT = this.computeTimeTop(
			pastMidnight + moment.duration(
				date - startOfDayDate.clone().stripTime()
			)
		);

		if (cTT == 0) {
			return this.computeTimeTop(
				86400000 + 
				moment.duration(
					date - startOfDayDate.clone().stripTime()
				)
			)
		}

		return this.computeTimeTop(
			pastMidnight + moment.duration(
				date - startOfDayDate.clone().stripTime()
			)
		);
	},
```

Right now I don't have much time to find a better fix but I'll try whenever I can if @arshaw is too busy and this is not fixed.

**Comment by pcstro** [comment]
Thank you! You got me in the right direction.
This is what solved it for me:

```
computeDateTop: function(date, startOfDayDate) {
		var startTime = moment.duration(date - startOfDayDate.clone().stripTime());

 		var cTT = this.computeTimeTop(startTime);


        //If an event is at the start of minTime
		if (startTime - this.minTime == 0) {
			return cTT;
		}

        //If an event gets dragged past midnight
        if (cTT == 0) {
            return this.computeTimeTop( 86400000 + startTime );
        }

       	return cTT;
},
```

**Comment by enekochan** [comment]
Thanks @pcstro ! I was just having an issue when an event was rendered in minTime and you already gave the solution ;)

**Comment by arshaw** [comment]
This should be fixed in **v3.3.0**
Blog post: https://fullcalendar.io/blog/2017/03/fullcalendar-33-and-scheduler-16/
